### ISSUE TITLE
[Feature suggestion] Workspace selector on boot by holding a button

### ISSUE THREAD

**Issue by blu-dev** [Raytwo/ARCropolis#133]
## What version are you on?
3.0.0

## Feature suggestion
ARCropolis can load plugins associated with mods.

### What would the feature do?
The feature would allow mod creators/distributors to package plugins with their mod in a easy-to-toggle fashion, whereas currently skyline plugins are not as easy to toggle.

### How should it be implemented?
The root of every mod can hold any amount of NRO files, which ARCropolis will attempt to load as plugins at runtime. Due to the limitations of file discovery, these plugins will have their entry/setup/main code run after file discovery has completed.

Whether or not a separate, runner thread will be running the plugins in parallel to file patching to speed things up has yet to be determined, however if you are developing plugins with the notion that you will be making use of this, it's best to make sure that any hooks/timing-dependent code will be planned accordingly.

### Is there an alternative way of doing things you'd like to suggest?
Alternatively, if Skyline were to include functionality to generate an NRR for new plugins, Skyline might be the one responsible for loading the plugins, but that is not a priority.

### Additional information
N/A
